mittab/templates/base/_navigation.html

```diff
@@ -97,6 +98,9 @@
             <a class="dropdown-item" href="/admin">Admin Interface</a>
           </div>
         </li>
+	<li class="nav-item">
+            <a class="nav-link" href="{{ logout }}">Logout</a>
```

Comment:
Nit: change indentation for consistency

```suggestion
        <a class="nav-link" href="{{ logout }}">Logout</a>
```